static/js/recent_topics_ui.js

```diff
@@ -620,6 +627,7 @@ function filter_buttons() {
 }
 
 export function hide() {
+    is_rt_visible = false;
```

Comment:
Can you put this on the line after `$("#recent_topics_view").hide();`, to make it clear this is tied to that?